src/main/io/gps.c

```diff
@@ -2637,6 +2639,30 @@ void GPS_calculateDistanceAndDirectionToHome(void)
     }
 }
 
+void GPS_calculateRelativePositionEarthGround(int32_t *homeLat, int32_t *homeLon, int32_t *targetLat, int32_t *targetLon, gpsRelativePos_t *gpsrelativePos) {
```

Comment:
GPS_distance_cm_bearing internally computes dLat and dLon in cm. Simply reuse that code (or split GPS_distance_cm_bearing into delta calculation and distance calculation)